<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YassMainWindow</class>
 <widget class="QMainWindow" name="YassMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>YASS</string>
  </property>
  <widget class="QWidget" name="centralFrame">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeWidget" name="buddyList">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <column>
       <property name="text">
        <string>Friends</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="linesFrame" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="dialFrame" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="dialBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Dial</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QComboBox" name="dialText">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="maxVisibleItems">
             <number>5</number>
            </property>
            <property name="maxCount">
             <number>15</number>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertBeforeCurrent</enum>
            </property>
            <property name="duplicatesEnabled">
             <bool>true</bool>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="linesTabs">
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="lineTab1">
          <attribute name="title">
           <string>Line 1</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextEdit" name="line1Text">
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>YASS</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionConfiguration"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHangup"/>
   <addaction name="actionAnswer"/>
   <addaction name="actionReject"/>
   <addaction name="actionXfer"/>
   <addaction name="actionHold"/>
   <addaction name="actionDtmf"/>
   <addaction name="actionHistory"/>
   <addaction name="actionVoicemail"/>
   <addaction name="actionMessage"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAnswer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar_icons.qrc">
     <normaloff>:/toolbar/icons/toolbar/answer.png</normaloff>:/toolbar/icons/toolbar/answer.png</iconset>
   </property>
   <property name="text">
    <string>answer</string>
   </property>
  </action>
  <action name="actionHangup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar_icons.qrc">
     <normaloff>:/toolbar/icons/toolbar/hangup.png</normaloff>:/toolbar/icons/toolbar/hangup.png</iconset>
   </property>
   <property name="text">
    <string>hangup</string>
   </property>
  </action>
  <action name="actionReject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar_icons.qrc">
     <normaloff>:/toolbar/icons/toolbar/reject.png</normaloff>:/toolbar/icons/toolbar/reject.png</iconset>
   </property>
   <property name="text">
    <string>reject</string>
   </property>
  </action>
  <action name="actionXfer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar_icons.qrc">
     <normaloff>:/toolbar/icons/toolbar/xfer.png</normaloff>:/toolbar/icons/toolbar/xfer.png</iconset>
   </property>
   <property name="text">
    <string>xfer</string>
   </property>
  </action>
  <action name="actionHold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar_icons.qrc">
     <normaloff>:/toolbar/icons/toolbar/hold.png</normaloff>:/toolbar/icons/toolbar/hold.png</iconset>
   </property>
   <property name="text">
    <string>hold</string>
   </property>
  </action>
  <action name="actionDtmf">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="toolbar_icons.qrc">
     <normaloff>:/toolbar/icons/toolbar/dtmf.png</normaloff>:/toolbar/icons/toolbar/dtmf.png</iconset>
   </property>
   <property name="text">
    <string>dtmf</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="icon">
    <iconset resource="toolbar_icons.qrc">
     <normaloff>:/toolbar/icons/toolbar/history.png</normaloff>:/toolbar/icons/toolbar/history.png</iconset>
   </property>
   <property name="text">
    <string>history</string>
   </property>
  </action>
  <action name="actionVoicemail">
   <property name="icon">
    <iconset resource="toolbar_icons.qrc">
     <normaloff>:/toolbar/icons/toolbar/voicemail.png</normaloff>:/toolbar/icons/toolbar/voicemail.png</iconset>
   </property>
   <property name="text">
    <string>voicemail</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionCall">
   <property name="text">
    <string>call</string>
   </property>
  </action>
  <action name="actionMessage">
   <property name="icon">
    <iconset resource="toolbar_icons.qrc">
     <normaloff>:/toolbar/icons/toolbar/message.png</normaloff>:/toolbar/icons/toolbar/message.png</iconset>
   </property>
   <property name="text">
    <string>message</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="toolbar_icons.qrc"/>
 </resources>
 <connections/>
</ui>
